pkg/packagekit/updates.jsx

```diff
@@ -327,7 +327,7 @@ class UpdateItem extends React.Component {
                         <i className="fa fa-fw" />
                     </td>
                     <th>{pkgsTruncated}</th>
-                    <td className="version">{info.version}</td>
+                    <td className="version"><span className="truncating">{info.version}</span></td>
```

Comment:
I thought the extra `<span>` would not be necessary, but turns out it is. With just adding the class to `<td>`, the bug icon still shines through the version text, with the span it's properly opaque.

Reply:
Yeah, I tried to get it to work without the span at first. It's needed, else we get into table-related wonkiness.